foundation/okfplugins/heading/models.py

```diff
@@ -20,6 +20,8 @@ class Heading(CMSPlugin):
         max_length=6, choices=HEADINGS_CHOICES, default="h1"
     )
     highlighted = models.BooleanField(default=False)
+    is_anchor = models.BooleanField(default=False)
+    anchor_id = models.CharField(max_length=200, blank=True)
```

Comment:
Do we need two fields? Is it possible to just check if `anchor_id` has a value?

Reply:
No. yes! I will remove the boolean field.